src/pages/ControllersIndex/_controllers.scss

```diff
@@ -43,10 +70,4 @@
     // cell.
     display: block !important;
   }
-
-  .controllers--registered-tooltip {
```

Comment:
I've changed this to use a normal tooltip component so this is no longer needed.